python/lsst/daf/butler/core/dimensions/standard.py

```diff
@@ -0,0 +1,411 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+__all__ = (
+)
+
+from types import MappingProxyType
+from typing import (
+    AbstractSet,
+    Dict,
+    Iterable,
+    Mapping,
+    Optional,
+    Set,
+)
+
+from .. import ddl
+from ..named import NamedValueAbstractSet, NamedValueSet
+
+from ._elements import Dimension, DimensionCombination, DimensionElement
+from ._topology import TopologicalFamily, TopologicalRelationshipEndpoint, TopologicalSpace
+from .construction import DimensionConstructionBuilder, DimensionConstructionVisitor
+
+
+class StandardTopologicalFamily(TopologicalFamily):
```

Comment:
That's a good point, but I think I'll wait for my current tickets to be further along before I rename, because I'm think the hierarchy of implementations will expand a bit and that may clarify what the different roles are a bit better.